<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TallennettuWidget</class>
 <widget class="QWidget" name="TallennettuWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>154</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tallennettu</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(170, 255, 127, 128);</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../pic/pic.qrc">:/pic/ok-128.png</pixmap>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>15</number>
     </property>
     <item>
      <widget class="QLabel" name="tunnisteellaLabel">
       <property name="text">
        <string>Tosite tallennettu tunnisteella</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="luonnosLabel">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Tosite tallennettu luonnoksena</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="tunnisteLabel">
       <property name="font">
        <font>
         <pointsize>26</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>15 / 19</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../pic/pic.qrc"/>
 </resources>
 <connections/>
</ui>
